src/gauss_fit.jl

```diff
@@ -3,19 +3,21 @@ export gauss_line_fit
 
 function gaussclossure(X::AbstractVector)
     function gaussloss(params)
-        a, σ = params[1], params[2]
+        b, a, σ = params[1], params[2], params[3]
         x = a * [exp(-(t)^2 / (2 * σ^2)) for t in axes(X, 1)]
+        x .+= b
```

Comment:
```suggestion
        x = a * [exp(-(t)^2 / (2 * σ^2)) + b for t in axes(X, 1)]
```
(Avoids a second allocation that way)